src/core/qgssimplifymethod.h

```diff
@@ -49,6 +49,11 @@ class CORE_EXPORT QgsSimplifyMethod
     //! Gets the tolerance of simplification
     inline double tolerance() const { return mTolerance; }
 
+    /** Sets the simplification threshold of the vector layer managed */
+    void setThreshold( float threshold ) { mThreshold = threshold; }
+    /** Gets the simplification threshold of the vector layer managed */
+    inline float threshold() const { return mThreshold; }
```

Comment:
It's not immediately clear what the difference between threshold and tolerance is here. This could get quite confusing in future if threshold is used in a different way by another provider. Can you expand these doc strings to describe exactly what the difference is?

(also, why float here?)